README.md

```diff
@@ -193,20 +193,8 @@ To access **Tooling for Apache Camel K** extension settings:
 
 Settings include:
 
-* **Proxy Namespace** - The namespace of the Kubernetes Proxy. You can select one of  two values: default and syndesis. The namespace is used to assemble the complete proxy URL and is part of the API definition in Kubernetes.
-* **Proxy Port** - he port of the Proxy URL to be used for accessing Kubernetes via Rest APIs. The default is **8000**.
-* **Proxy URL** - The server proxy URL for your Kubernetes service. The default is **http://localhost**. You can change it to any appropriate service URL. This value is combined with the port to construct URLs at runtime when starting the proxy and using proxy calls.  For details on how to create the local proxy (i.e. 'kubectl proxy –port=8000'), see [Use an HTTP Proxy to Access the Kubernetes API)[https://kubernetes.io/docs/tasks/access-kubernetes-api/http-proxy-access-api/] and [Starting a local Kubernetes proxy](#starting-a-local-kubernetes-proxy).)
 * **Show Status Bar Messages** - Indicates whether to show messages in the status bar to indicate when the system is updating, such as when the Camel K Integrations view is being refreshed or a new Integration is being deployed.
-* **Use Proxy** - Determines whether the Camel K Integrations view retrieves the list of running integrations by using the local 'kubectl' application or by using the Kubernetes Rest API and calls through the Proxy URL/Namespace combination above. The ultimate URL becomes [proxyurl]/apis/camel.apache.org/v1alpha1/namespaces/[namespace]/integrations.
-
-## Starting a local Kubernetes proxy
-
-**Note:** This option is only available if you have the Minikube executable installed.
-
-1. In In the VS Code editor, open the command palette by pressing **Ctrl+Shift+P** or **F1**. 
-2. From the options, select **Apache Camel K: Start the Kubernetes proxy server**.
-
-A new Kubernetes proxy starts. It uses the Minikube executable (kubectl proxy --port=8000) and refreshes the Apache Camel K Integrations view immediately. This command creates a local proxy at the URL composed of the Proxy URL and Proxy Port set in the settings. For example, with the defaults, the proxy URL becomes **http://localhost:8000**.
+* **Namespace** - The namespace of the Kubernetes Proxy. You can select one of  two values: default and syndesis. The namespace is used to assemble the complete proxy URL and is part of the API definition in Kubernetes.
```

Comment:
it is kubernetes namespace? or another kind of namespace? we can select only "default" or "syndesis"? why no other?
If Kubernetes namespaces, i think that by default it is installing in the current namespace, so inside Che by default it is in "che" workspace

Reply:
I fixed it with a follow-up PR - https://github.com/camel-tooling/vscode-camelk/pull/169/files